app/gui2/src/components/GraphEditor.vue

```diff
@@ -581,6 +591,7 @@ const groupColors = computed(() => {
       :allowNavigationLeft="stackNavigator.allowNavigationLeft.value"
       :allowNavigationRight="stackNavigator.allowNavigationRight.value"
       :zoomLevel="100.0 * graphNavigator.targetScale"
+      :componentsSelected="nodeSelection.selected.size"
```

Comment:
At this point it would probably make sense to make `TopBar` actually use the context variables directly, instead of passing so much data through props and events. Doesn't have to be done in this PR, but it is worth to clean it up at some point to cut down on `GraphEditor` rerenders.